components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/IdentityManagementServiceUtil.java

```diff
@@ -123,6 +124,7 @@ public void init() {
                     .APP_PASSWORD).toCharArray();
             String serviceContextURL = properties
                     .getProperty(IdentityManagementEndpointConstants.ServiceConfigConstants.SERVICE_CONTEXT_URL);
+            contextURL = serviceContextURL;
             this.serviceContextURL = StringUtils.isBlank(serviceContextURL) ? IdentityUtil.getServerURL(
                     IdentityUtil.getServicePath(), true, true) : serviceContextURL;
```

Comment:
Is this kept for the legacy behavior

Reply:
Yes. This is only used by the deprecated util method.